src/Test/Perf/util/test_util.csx

```diff
@@ -255,28 +262,42 @@ void Log(string info)
     }
 }
 
-/// Logs the result of a finished process
+// Logs an error message.
+//
+// The actual implementation of this method may change depending on
+// if the script is being run standalone or through the test runner.
+void LogError(string info)
```

Comment:
Basically, I want stdout for the child process to be printed to stdout for the parent and stderr for the child process to be printed to stderr for the parent.  The reason is that some runners/CI systems treat writing to stderr as a failure and report that text in a more prominent way (Team Build is one, but I imagine others might have similar behavior).  I expect the messages written to each to be distinct (not writing the same message twice).